nixos/modules/services/hardware/nvidia-container-toolkit/cdi-generate.nix

```diff
@@ -37,9 +37,9 @@ writeScriptBin "nvidia-cdi-generator" ''
     ${lib.getExe' nvidia-container-toolkit "nvidia-ctk"} cdi generate \
       --format json \
       --device-name-strategy ${deviceNameStrategy} \
-      --ldconfig-path ${lib.getExe' glibc "ldconfig"} \
+      --ldconfig-path ${lib.getExe' glibc.bin "ldconfig"} \
```

Comment:
Huh, I thought `getExe` would select `.bin` automatically?

Reply:
You are right, will update